contrib DocToken parser bug (`@return` vs `@returns`)

https://github.com/scalameta/scalameta/blob/eeb1ea6e62aea54dee4cab42bd91849704827004/scalameta/contrib/shared/src/main/scala/scala/meta/contrib/DocToken.scala#L146

```diff
-  case object Return extends TagKind("@returns", numberParameters = 1)
+  case object Return extends TagKind("@return", numberParameters = 1)
```

🤔 

`build.sbt`

```scala
libraryDependencies += "org.scalameta" %% "scalameta" % "4.9.2"

scalaVersion := "2.13.13"
```

`Main.scala`

```scala
import scala.meta._
import scala.meta.contrib.DocToken

object Main {
  def parse(src: String): Option[List[DocToken]] = {
    scala.meta.contrib.CommentOps.docTokens(
      new scala.meta.tokens.Token.Comment(
        input = Input.String(src),
        dialect = dialects.Scala3,
        start = 0,
        end = src.length,
        value = src
      )
    )
  }

  def main(args: Array[String]): Unit = {
    println(parse("""
      /**
       * @return xxx
       */
      """))
    // Some(List(Description(@return xxx)))

    println(
      parse("""
      /**
       * @returns xxx
       */
      """)
    )
    // Some(List(Return(xxx)))
  }
}
```


```
$ scaladoc --version
Scaladoc version 2.13.13 -- Copyright 2002-2024, LAMP/EPFL and Lightbend, Inc.
$ cat A.scala 
trait A {
  /**
   * @return xxx
   */
  def x: Int

  /**
   * @returns yyy
   */
  def y: Int
}
$ scaladoc A.scala                      
A.scala:7: warning: Tag '@returns' is not recognised
  /**
  ^
1 warning
```


### Scala 3.3.3

<img width="309" alt="scala-3" src="https://github.com/scalameta/scalameta/assets/389787/8ca7b10b-0375-4e0f-b9df-2c879dc29305">


### Scala 2.13.13
<img width="315" alt="scala-2-13" src="https://github.com/scalameta/scalameta/assets/389787/438f4036-88df-4dfa-a2ba-afa6e34975d4">


### [in scala2 repo](https://github.com/scala/scala/tree/05533ed20c14129a3b7fdd6fb23f131b3f2cded5/) git grep result

```
$ git grep " @return " | wc -l
    1474
$ git grep " @returns " | wc -l
       0
```

--- a/scalameta/contrib/shared/src/main/scala/scala/meta/contrib/DocToken.scala
+++ b/scalameta/contrib/shared/src/main/scala/scala/meta/contrib/DocToken.scala
@@ -143,7 +143,7 @@ object DocToken {
   /**
    * Documents the return value of a method.
    */
-  case object Return extends TagKind("@returns", numberParameters = 1)
+  case object Return extends TagKind("@return", numberParameters = 1)
 
   /**
    * Documents an exception type that may be thrown by a method or class constructor.

